README.md

```diff
@@ -23,6 +23,6 @@ Data are not included here. Expects a bunch of files in TSV format generated
 with the help of [this scraper](https://github.com/rodzam/ncaab-stats-scraper),
 to be living in the (now empty) `data` directory.
 
-#### Spark
+#### Spark  
```

Comment:
please configure your editor to strip trailing whitespace from lines! (makes for cleaner diffs, since they won't be cluttered with comparisons like this which are effectively useless.)